packages/bpk-component-text/src/BpkText.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './BpkText.scss';
+
+const getClassName = cssModules(STYLES);
+
+const TYPES = ['xs', 'sm', 'base', 'lg', 'xl', 'xxl'];
+
+// FIXME: Should we prefer `Map` or Object?
```

Comment:
Yeah the only reason I favoured `Map` is because it has a constructor that takes key, value pairs and that can easily be constructued with `map` when the value is derivable from the key